pkgs/applications/misc/snapmaker-luban/default.nix

```diff
@@ -0,0 +1,23 @@
+{ branch ? "stable", pkgs }:
```

Comment:
You're using `callPackage` so you can replace `pkgs` with `callPackage` and `fetchurl`

Reply:
Fixed